system/inc/system_cloud.h

```diff
@@ -56,9 +56,43 @@ void system_set_time(time_t time, unsigned param, void* reserved);
 
 typedef enum
 {
-	BOOLEAN = 1, INT = 2, STRING = 4, DOUBLE = 9
```

Comment:
Is this backward compatible?  These enums are what users write in their code, aren't they?